dissect/target/plugins/os/windows/regf/runkeys.py

```diff
@@ -65,7 +63,7 @@ def runkeys(self):
             domain (string): The target domain.
             ts (datetime): The registry key last modified timestamp.
             name (string): The run key name.
-            path (string): The run key path.
+            path (string): The run key command.
```

Comment:
I would suggest to then also change the record name. But maybe it's better to do that with the full fix.

So long story short, I would suggest to keep it path here.